csrc/scheduler/utils.cpp

```diff
@@ -1386,9 +1382,9 @@ IterDomain* projectIdToRoot(
 }
 
 // Take the inner most root id from innerMostAllocDim and project it to the
-// logical domain if the provided domain is on the logical domain. If vectorize,
-// will not project if not following the inner most path.
-IterDomain* projectIdToRFactor(
+// allocation domain if the provided domain is on the allocation domain. If
```

Comment:
Should this comment be: "if the provided ~domain~ reference_id is on the allocation domain"